source/extensions/filters/network/zookeeper_proxy/filter.h

```diff
@@ -121,7 +121,7 @@ class ZooKeeperFilter : public Network::Filter,
                         DecoderCallbacks,
                         Logger::Loggable<Logger::Id::filter> {
 public:
-  explicit ZooKeeperFilter(ZooKeeperFilterConfigSharedPtr config);
+  explicit ZooKeeperFilter(ZooKeeperFilterConfigSharedPtr config, TimeSource& time_source);
```

Comment:
no longer needs `explicit` (also looking at getting [this clang-tidy check](https://clang.llvm.org/extra/clang-tidy/checks/google-explicit-constructor.html) enabled